tardis/tests/tests_slow/report.py

```diff
@@ -32,10 +32,12 @@ def __init__(self, dokuwiki_details):
                     password=dokuwiki_details["password"])
             except gaierror, dokuwiki.DokuWikiError:
                 self.doku_conn = None
-                print "Dokuwiki connection could not be established!"
         else:
             self.doku_conn = None
 
+        # This variable will be set True when report gets uploaded on Dokuwiki.
+        self.is_report_uploaded = False
```

Comment:
I wouldn't use a variable but check in sessionfinish that the page is indeed accessible.